addons/point_of_sale/views/point_of_sale.xml

```diff
@@ -12,6 +12,7 @@
 
             <script type="text/javascript" src="/point_of_sale/static/lib/backbone/backbone.js"></script>
             <script type="text/javascript" src="/point_of_sale/static/lib/waitfont.js"></script>
+            <script type="text/javascript" src="/point_of_sale/static/lib/FileSaver.js"></script>
```

Comment:
Do we really need another js library, can't we reimplement the few needed methods? (real question)

Reply:
no, it is used and I think it's ok to import it as is.
https://github.com/odoo/odoo/pull/29427/files#diff-1a22c54c97375e3b10e599c85cc05bddR770